app/uk/gov/hmrc/gform/form/FormService.scala

```diff
@@ -64,10 +63,7 @@ class FormService(save4Later: Save4Later) {
           status = newStatus(form, userData.formStatus),
           visitsIndex = userData.visitsIndex,
           obligations = userData.obligations)
-      _ <- {
-        Logger.debug(Json.prettyPrint(Json.toJson(newForm)) + "UpdateUserData")
```

Comment:
was this meant to be removed?

Reply:
Yes. It was added as a way to check what was actually reaching the backend when a call was made. After the issue was resolved it was removed